### PR TITLE
Remove DefinePlugin to fix undefined SDK_VERSION

### PR DESCRIPTION
**Pull request by lboyette-okta**
Resolves: OKTA-94206

This fixes an issue where simply calling 'require' on the npm version of the sdk causes SDK_VERSION and other globals variables to be undefined. Webpack usually defines these variables using the DefinePlugin; now we build a config that can be 'require'd without webpack.

@rchild-okta 
@alexeystadnikov-okta

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 /node_modules
 Gruntfile.js
 *.config.js
+writeConfig.js
```
Comment (rchild-okta) [inline .eslintignore]: nit: Why do we need to add this to eslintignore? Is it just for the console line (and if so, why not just disable that check in the file)?

File: package.json [hunk 2]
```diff
@@ -49,6 +50,7 @@
     "grunt-contrib-jasmine": "0.9.2",
     "grunt-shell": "1.3.0",
     "jquery": "2.1.4",
+    "json-loader": "0.5.4",
```
Comment (rchild-okta) [inline package.json]: Where is this used?

File: test/spec/general.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-/* globals SDK_VERSION */
 define(function(require) {
   var OktaAuth = require('OktaAuth');
   var Q = require('q');
   var util = require('../util/util');
   var _ = require('lodash');
+  var packageJson = require('../../package.json');
```
Comment (rchild-okta) [inline test/spec/general.js]: Can we load config.json here (seems like we're running the writeconfig task before test as well)?

**Comment by rchild-okta** [comment]
Couple small things, but overall lgtm after fixing 🚀

File: test/spec/general.js (hunk 3 above)
Comment (lboyette-okta) [inline test/spec/general.js]: We are, but this verifies that the config.json is written with the correct values.

File: .eslintignore (hunk 1 above)
Comment (lboyette-okta) [inline .eslintignore]: I'll disable it in the file. It's just for the console

**Pull request merged**